atlp-rwanda/falcons-ec-be
# #184637273 payment using Stripe API

#### What does this PR do?

- adds stripe API for payment

#### Description of Task to be completed?

The backend needs to handle the payment process via Stripe API, including processing the payment, updating the order status, and sending the confirmation to the front end.

#### How should this be manually tested?

- add items to the cart
- on success, you should get a 200 status code and a url to fill in card details
- and on failure, you get a 400 status code 

#### Any background context you want to provide?

- use card number: 4242424242424242, expiry month: 12, expiry year: [some future year, eg: 34], cvc: 123

#### What are the relevant pivotal tracker/Trello stories?
 Tracker: 184637273 
#### Screenshots (if appropriate)
 none
#### Questions:
 none
